### PR TITLE
function find-word

### PR DESCRIPTION
**Pull request by Flotss**

### CODE REVIEW

File: functions/api/motus/find-word.js [hunk 1]
```diff
@@ -1,90 +1,92 @@
...
       }
+
+      if (letterOut.length === 0) {
+        letterOut = new RegExp('!.*')
+      } else {
+        letterOut = new RegExp('[' + letterOut + ']')
+      }
+
+      words.forEach(word => {
+        word = word.normalize("NFD").replace(/\p{Diacritic}/gu, "")
```
Comment (MAXOUXAX) [inline functions/api/motus/find-word.js:76]: Are you sure it works to remove accents?

**Review by MAXOUXAX — CHANGES_REQUESTED** [review]
Please check if accents are properly removed from words.

File: functions/api/motus/find-word.js (hunk 1 above)
Comment (Flotss) [reply functions/api/motus/find-word.js:76]: Yes, It's work but your processor burn

Comment (MAXOUXAX) [reply functions/api/motus/find-word.js:76]: Maybe you could try the way I did it in 
https://github.com/MAXOUXAX/maxouxax.me/blob/develop/functions/api/motus/get-random-word.js#L16

Comment (Flotss) [reply functions/api/motus/find-word.js:76]: It's the same thing I think

**Pull request merged**